git_parser.py

```diff
@@ -39,7 +36,7 @@ def show_commit(output_file,repository):
 
     Row_number += 1
 
-    for commit in Repository(path_to_repo=repository, since=firstDay, to=toDay).traverse_commits():
```

Comment:
Do not remove date use.
Instead lets add new flag call '--show_lats_days' it will get the number of days I want to parse commits from today and backwards.

Example: --show_lats_days=15  will show me only commits from the last 15 days.
if not provided - show all commits In the repository.